packages/atomic/src/components/search/facets/atomic-facet/atomic-facet.tsx

```diff
@@ -112,6 +112,16 @@ export class AtomicFacet implements InitializableComponent, BaseFacet<Facet> {
    * Possible values are 'score', 'alphanumeric', 'occurrences', and 'automatic'.
    */
   @Prop({reflect: true}) public sortCriteria: FacetSortCriterion = 'automatic';
+  /**
+   * Specifies a custom order by which to sort the facet values.
+   *
+   * Example:
+   * ```html
+   * <!-- Sort @ticketstatus facet values in a logical order for support tickets -->
+   * <atomic-facet field="@ticketstatus" custom-sort="New,Opened,Feedback,Resolved"></atomic-facet>
+   * ```
+   */
+  @Prop({reflect: true}) public customSort?: string;
```

Comment:
Instinctively, this sounds more like "pinning" values to me. Additionally, I find it unclear what happens to values that aren't included in the custom sort. Are they hidden/removed? Are they sorted alphanumerically?

Reply:
I recall that Stencil supports arrays of strings. Do they split on commas?

Reply:
customSort is how it's called in the API. Not pinning.

> Additionally, I find it unclear what happens to values that aren't included in the custom sort. Are they hidden/removed? Are they sorted alphanumerically?

Specified values in customSort will appear first in the facet. Then the API will fill remaining values until `numberOfValues` is reached using either score or alphanumeric, depending on the state of the facet (expanded, collapsed).

> I recall that Stencil supports arrays of strings. Do they split on commas?

This is mimicking the same way we do it for all other such option in Atomic. Either we change it everywhere, or we keep the same pattern.

And the old pattern has served us very well so far, so I see no reason to invest time to change it.